arboretum/permissions/fetchers/github/fetch_org_collaborators.py

```diff
@@ -0,0 +1,60 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Github organization collaborators fetcher."""
+
+import json
+from urllib.parse import urlparse
+
+from arboretum.permissions.evidences.github.gh_org_collaborators import (
+    GhOrgCollaboratorsEvidence
+)
+
+from compliance.evidence import DAY
+from compliance.fetch import ComplianceFetcher
+from compliance.utils.services.github import Github
+
+
+class GithubOrgCollaboratorsFetcher(ComplianceFetcher):
```

Comment:
This fetcher should be refactored to use the [raw_evidence context manager](https://github.com/ComplianceAsCode/auditree-framework/blob/main/compliance/evidence.py#L283-L308).  You should also follow the pattern of adding the evidence to the evidence cache rather than how you're handling it right now.  See [here](https://github.com/ComplianceAsCode/auditree-arboretum/blob/main/arboretum/issue_mgmt/fetchers/fetch_github_issues.py#L37-L63).

See doc [ref](https://complianceascode.github.io/auditree-framework/design-principles.html#evidence-validation).